### PR TITLE
Admin user profile page

### PR DESCRIPTION
**Pull request by lrs8810**
(#49 ) Admin can see a user profile page but cannot edit the profile or password

### CODE REVIEW

File: app/views/admin/users/show.html.erb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<section id='user-info'>
+  <%=
+    render partial: 'partials/user_info', locals: {
+      name: @user.name,
+      address: @user.address,
+      city: @user.city,
+      state: @user.state,
+      zip: @user.zip,
+      email: @user.email
+    }
```
Comment (CoopTang) [inline app/views/admin/users/show.html.erb:10]: Great reuse of our partials!

**Review by CoopTang — APPROVED** [review]
Looks great!

File: app/views/admin/users/show.html.erb (hunk 1 above)
Comment (freeheeling) [reply app/views/admin/users/show.html.erb:10]: They are handy.

**Review by freeheeling — APPROVED** [review]
Approved.

**Pull request merged**